docs/core/compatibility/core-libraries/7.0/obsolete-apis-with-default-diagnostic.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Breaking change: .NET 7 obsoletions with default diagnostic ID"
+titleSuffix: ""
+description: Learn about the .NET 7 breaking change in core .NET libraries where some APIs have been marked as obsolete with the default diagnostic ID.
+ms.date: 10/19/2022
+---
+# API obsoletions with default diagnostic ID (.NET 7)
+
+Several [APIs](#affected-apis) have been marked as obsolete in .NET 7. Referencing these APIs in your code will result in build warnings. In C#, the compiler diagnostic for these obsoletions is [CS0618](../../../../csharp/language-reference/compiler-messages/cs0618.md).
+
+## Previous behavior
+
+Previously, the affected APIs could be referenced without any build warnings.
+
+## New behavior
+
+Starting in .NET 7, referencing the affected APIs will result in build warnings.
+
+## Version introduced
+
+.NET 7 Preview 3
+
+## Type of breaking change
+
+These obsoletions can affect [source compatibility](../../categories.md#source-compatibility).
+
+## Reason for change
+
+These APIs were previously marked obsolete in the implementation assemblies but not in the [reference assemblies](../../../../standard/assembly/reference-assemblies.md). The reference assemblies have now been updated to match the implementation assemblies.
+
+## Recommended action
+
+Follow the recommended action that's emitted when you use the obsolete API.
+
+## Affected APIs
```

Comment:
I omitted the following two from the issue because they were already marked obsolete from what I can tell.

- System.Net.HttpWebRequest.HttpWebRequest(SerializationInfo, StreamingContext)
- System.Net.HttpWebRequest.GetObjectData(*)